airflow/version.py

```diff
@@ -18,4 +18,4 @@
 # under the License.
 #
 
-version = '1.10.15'
+version = '1.10.15+siftpatch7a1'
```

Comment:
This patching is getting fancy...

Reply:
I normally use the PyPi alpha sequencing `a1, a2` etc to publish pre-release versions to PyPi before merging to make sure that it can be packed and installed etc.

@michelgoldstein - question here though. Should I reset the `siftpatch` number since the base version is different or just continue with the next patch number which would be `siftpatch7`?